components/unstoppable_domains/pref_names.h

```diff
@@ -0,0 +1,15 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_UNSTOPPABLE_DOMAINS_PREF_NAMES_H_
+#define BRAVE_COMPONENTS_UNSTOPPABLE_DOMAINS_PREF_NAMES_H_
+
+namespace unstoppable_domains {
+
+extern const char kResolveMethod[];
```

Comment:
pref names / constants / etc could be `constexpr char kSomething[] = "...";` defined right in the header file. Saves few extra `.cc`

Reply:
Fixed in https://github.com/brave/brave-core/pull/7702/commits/81f500874e38a1f91885cfce5899ac3c0aebbc87